org-code-javabuilder/lib/src/main/java/org/code/javabuilder/JavaProjectFile.java

```diff
@@ -43,10 +43,14 @@ public void setFileName(String fileName) {
 
   @Override
   public void setFileContents(String fileContents) {
-    this.fileContents = fileContents;
+    this.fileContents = this.importSystem(fileContents);
   }
 
   public void setClassName(String className) {
     this.className = className;
   }
+
+  private String importSystem(String fileContents) {
+    return "import org.code.lang.System;\n" + fileContents;
```

Comment:
Should we check if this import already exists before adding it (especially in the case of `setFileContents`?)

Reply:
good idea

Reply:
I added a check if the file contents start with the import we won't add it. I just learned you actually can have duplicate imports of the same package and Java doesn't care, so even if we end up double-importing student's won't see an issue. I wanted to only check startsWith in the case that for some reason the import is buried in a string somewhere.

Reply:
ended up switching this back because it felt overly complicated for limited benefit.

Reply:
Will this cause compiler errors to always be off-by-one?

Reply:
added a fix for this!